playbooks/setup_forklift.yml

```diff
@@ -99,7 +99,19 @@
       when:
         - forklift_install_pulp_from_galaxy
         - pipeline_version is defined
-        - pipeline_version == 'nightly' or pipeline_version is version('3.18', '>=')
+        - pipeline_version != 'nightly' or pipeline_version is version('3.18', '>=')
```

Comment:
!= nightly and >= 3.18 and < 3.39 please

Reply:
It's better to keep `- pipeline_version != 'nightly' or pipeline_version is version('3.18', '>=') and pipeline_version is version('3.39', '<')`

or 

`- pipeline_version != 'nightly' or pipeline_version is version('3.18', '>=')`
`- pipeline_version is version('3.39', '<')

Reply:
I think the following is most correct and easy to the eyes:
```
- pipeline_version != 'nightly'
- pipeline_version is version('3.18', '>=')
- pipeline_version is version('3.39', '<')
```

Reply:
Done